examples/context-example.php

```diff
@@ -118,15 +118,16 @@ function matchContextDemo($authProvider, $projectId, $contextUid, $fileUri)
  * @param \Smartling\AuthApi\AuthApiInterface $authProvider
  * @param string $projectId
  * @param string $fileUri
+ * @param $contentFileUri
```

Comment:
Can you add description because even for me it's not clear what's the difference between `fileUri` and `contentFileUri`

Reply:
Done.